internal/fetch_config.go

```diff
@@ -6,32 +6,38 @@ import (
 )
 
 func NewFetchConfig(backupName,dbDataDirectory string, folder storage.Folder, spec *TablespaceSpec,
```

Comment:
go fmt

Reply:
Fixed in https://github.com/wal-g/wal-g/pull/687/commits/0b2c8f6d2e118d0ed0d05ef1c711d5ba93ea089f